src/Queue/Connectors/RabbitMQConnector.php

```diff
@@ -42,6 +42,7 @@ public function connect(array $config): Queue
 
         $queue = $this->createQueue(
             Arr::get($config, 'worker', 'default'),
+            Arr::get($config, 'custom_worker_type', 'default'),
```

Comment:
This config item does not follow other connection properties available to all drivers. 
Rather put this in the options property [ options => [ 'custom_option' => 'value' ]]